be/src/storage/rowset/column_iterator.cpp

```diff
@@ -64,4 +64,28 @@ Status ColumnIterator::fetch_dict_codes_by_rowid(const Column& rowids, Column* v
     return fetch_dict_codes_by_rowid(p, rowids.size(), values);
 }
 
+Status ColumnIterator::next_batch(const SparseRange<>& range, Column* dst) {
```

Comment:
Add default implementation of `next_batch`, the original implementation was return `NotSupported` error.